resources/js/i18n/en.json

```diff
@@ -808,5 +840,12 @@
                 }
             }
         }
+    },
+    "keyboard": {
+        "tab": "Tab",
+        "space": "Spacebar"
+    },
+    "storage": {
+        "save_to_local_toggle": "Enable if you want to store the values locally"
     }
```

Comment:
not used

Reply:
Fixed in e1f017ea68742e1c0d73bf36848c7e753edfb3c9